tox.ini

```diff
@@ -20,6 +21,13 @@ deps =
 commands =
     black --check semeio tests setup.py
 
+[testenv:flake8]
+description = Check PEP8 using flake8
+deps =
+    flake8
+commands =
+    flake8 semeio tests setup.py
```

Comment:
I´m fine with having `flake8` in the test suite, but _think_ we should do `python -m flake8 ...`, that goes for the other tests as well, but that is for another issue.